raster/scripts/python/genraster.py

```diff
@@ -26,15 +26,16 @@
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 ###############################################################################
+from __future__ import print_function
```

Comment:
This import and similar below are not needed, as I don't see any use of `print()` function [as documented](https://docs.python.org/3/library/functions.html#print).

Reply:
This directive will [prevent compatibility regressions](https://docs.python.org/3/howto/pyporting.html#prevent-compatibility-regressions) by changing Python 2 so that it is a syntax error to use legacy __print__ statements.  This prevents Python 2 developers from breaking Python 3 compatibility.

Reply:
these imports are fine and can stay. Python 2 [will not be maintained past 2020](https://pythonclock.org/), so I won't expect too many Python 2 developers down the road.